tensorflow_cloud/run.py

```diff
@@ -39,6 +39,7 @@ def run(
     entry_point_args=None,
     stream_logs=False,
     docker_image_bucket_name=None,
+    labels = {},
```

Comment:
We will need to change the name to reflect what it is. `labels` is vague. May be `job_labels`?

Reply:
Sounds good. Changed all `labels` -> `job_labels`